packages/at_onboarding_cli/pubspec.yaml

```diff
@@ -21,15 +21,28 @@ dependencies:
    path: ^1.8.1
    zxing2: ^0.2.0
    at_auth: ^1.0.3
-   at_chops: ^1.0.6
-   at_client: ^3.0.69
-   at_commons: ^3.0.58
-   at_lookup: ^3.0.43
+   at_chops: ^1.0.7
+   at_client: ^3.0.73
+   at_commons: ^4.0.0
+   at_lookup: ^3.0.44
    at_server_status: ^1.0.3
-   at_utils: ^3.0.15
+   at_utils: ^3.0.16
+
+dependency_overrides:
+  at_server_status:
+    git:
+      url: https://github.com/atsign-foundation/at_libraries.git
+      ref: trunk
+      path: packages/at_server_status
+
+  at_auth:
```

Comment:
check with @purnimavenkatasubbu for PR in at_auth. publish and merge

Reply:
updated